Library/Formula/gpac.rb

```diff
@@ -12,14 +12,11 @@ class Gpac < Formula
   homepage 'http://gpac.wp.mines-telecom.fr/'
 
   stable do
-    url 'https://downloads.sourceforge.net/gpac/gpac-0.5.0.tar.gz'
-    sha1 '48ba16272bfa153abb281ff8ed31b5dddf60cf20'
-
-    # Fixes build against ffmpeg 2.x; backported from upstream SVN
-    patch :DATA
```

Comment:
If you're removing the patch, can you also delete `__END__` and all the lines following it?